src/main/java/com/laba/solvd/jdbc/model/Sale.java

```diff
@@ -0,0 +1,87 @@
+package com.laba.solvd.jdbc.model;
+
+import java.util.Date;
+import java.util.Objects;
+
+public class Sale {
+    private int saleId;
+    private double totalPrice;
```

Comment:
please follow same formatting in all classes:
```suggestion
public class Sale {

    private int saleId;
    private double totalPrice;
```